src/app/pages/search-listing/search-listing.component.html

```diff
@@ -57,6 +57,16 @@
               {{ resultSet?.totalResults }} results found
             </ng-container>
           </div>
+          <div>
```

Comment:
Would it be possible to align this control closer to the "Sort by" control? My feeling is the sort and page size controls should act as a related group.

![image](https://user-images.githubusercontent.com/3219480/223703203-9dd829e8-d9cb-4cf4-a0b6-a1e01fc93db3.png)

The inclusion of the page size control is welcome though.

Reply:
Agreed, done